omod/src/main/java/org/openmrs/module/appointments/web/contract/DailyAppointmentServiceSummary.java

```diff
@@ -9,13 +9,23 @@ public class DailyAppointmentServiceSummary {
     private Date appointmentDate;
     private String appointmentServiceUuid;
 
+    private String appointmentServiceName;
+
     public DailyAppointmentServiceSummary(Date appointmentDate, String appointmentServiceUuid, Integer allAppointmentsCount, Integer missedAppointmentsCount) {
         this.allAppointmentsCount = allAppointmentsCount;
         this.missedAppointmentsCount = missedAppointmentsCount;
         this.appointmentDate = appointmentDate;
         this.appointmentServiceUuid = appointmentServiceUuid;
     }
 
+    public DailyAppointmentServiceSummary(Date appointmentDate, String appointmentServiceUuid, Integer allAppointmentsCount, Integer missedAppointmentsCount, String appointmentServiceName) {
+        this.allAppointmentsCount = allAppointmentsCount;
+        this.missedAppointmentsCount = missedAppointmentsCount;
+        this.appointmentDate = appointmentDate;
+        this.appointmentServiceUuid = appointmentServiceUuid;
+        this.appointmentServiceName = appointmentServiceName;
```

Comment:
Why do we need the `appointmentServiceName` if we have the `appointmentServiceUuid`?